qemu/tests/blockdev_snapshot_merge.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from provider.blockdev_snapshot_base import BlockDevSnapshotTest
+
+
+class BlockdevSnapshotMergeTest(BlockDevSnapshotTest):
```

Comment:
Just fyi.
A little complicated for the code implementation, maybe next time you could  try blockdev_commit_base.BlockDevCommitTest, which includes multiple snapshots.
Yours is OK for me.

Reply:
In this case, we do commit offline. Although it includes multiple snapshots in blockdev_commit_base.BlockDevCommitTest, I still prefer provider.blockdev_snapshot_base, thanks for your advice.